docs/site/tutorials/todo/todo-tutorial-model.md

```diff
@@ -58,8 +58,8 @@ Enter an empty property name when done
 ? Enter the property name: id
 ? Property type: number
 ? Is id the ID property? Yes
-? Is it required?: No
 ? Is id generated automatically? No
+? Is it required?: No
 ? Default value [leave blank for none]:
 
 Let's add another property to Todo
```

Comment:
I notice that we don't have code snippet or links to the file of the generated model or the repository in https://loopback.io/doc/en/lb4/todo-list-tutorial-repository.html. Do we want to add them?